docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -118,15 +118,6 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td class="yes">Yes</td>
       <td class="yes">Yes</td>
     </tr>
-    <tr>
```

Comment:
Agreed on mentioning access to SRM. I'm also adding a footnote to clarify that users can see items originating from Jira and Codacy repos that they follow.

About the operation, we already mention the limitation on the SRM page. Since it's only a "passive" action, I wouldn't use up a full row to mention it (the others are active, org-level management actions: "configure", "manage")